overviewer_core/aux_files/genPOI.py

```diff
@@ -220,9 +233,13 @@ def load_cache(cls, outputdir):
     @classmethod
     def save_cache(cls, outputdir):
         cache_file = os.path.join(outputdir, "uuidcache.dat")
-        gz = gzip.GzipFile(cache_file, "wb")
-        json.dump(cls.uuid_cache, gz)
-        logging.info("Wrote UUID cache with %d entries", len(cls.uuid_cache.keys()))
+        try:
+            gz = gzip.GzipFile(cache_file + ".tmp", "wb")
+            json.dump(cls.uuid_cache, gz)
+            os.rename(cache_file + ".tmp", cache_file)
```

Comment:
no, I'm talking about files.py in overviewer_core... which, uh... I think you wrote.